sklearn/neighbors/tests/test_lof.py

```diff
@@ -32,8 +34,7 @@
 iris.target = iris.target[perm]
 
 
-@ignore_warnings(category=DeprecationWarning)
-# contamination changed to 'auto' 0.22
+@pytest.mark.filterwarnings('ignore:default contamination parameter 0.1')
```

Comment:
Should we be giving the version number of scikit-learn where this line should be remove: "# 0.22"?